SignalMessaging/utils/ConversationStyle.swift

```diff
@@ -136,76 +136,131 @@ public class ConversationStyle: NSObject {
 
     private static let defaultBubbleColorIncoming = UIColor.ows_messageBubbleLightGray
 
-    // TODO:
     @objc
-    public let bubbleColorOutgoingUnsent = UIColor.ows_red
+    public let bubbleColorOutgoingUnsent = UIColor.gray
```

Comment:
Just checking because I'm surprised this is the default UIColor.gray, rather than some specified hex value.

Reply:
Yup, these are the "old" JSQ values - for now.

Reply:
No, you're right, this changed in Zeplin today.